package/MDAnalysis/analysis/contacts.py

```diff
@@ -143,9 +143,9 @@
     def is_any_closer(r, r0, dist=2.5):
         return np.any(r < dist)
 
-The first two parameters `r` and `r0` are provided by :class:`Contacts` when it
+The first two parameters ``r`` and ``r0`` are provided by :class:`Contacts` when it
```

Comment:
Actually, *single backticks* are mandated in the [NumPy doc style](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections) for references to variable names (scroll down to *4. Parameters*).